.kokoro/presubmit/prerelease-deps.cfg

```diff
@@ -5,3 +5,9 @@ env_vars: {
     key: "NOX_SESSION"
     value: "prerelease_deps"
 }
+
+# Credentials needed to test universe domain.
+env_vars: {
+  key: "SECRET_MANAGER_KEYS"
+  value: "client-library-test-universe-domain-credential"
+}
```

Comment:
I'm wondering if we've considered running `prerelease-deps` in continuous instead of presubmits. This increases presubmit running time and the extra two runs of the system test suite are more prone to 429 errors and server-side resource overload etc.

Reply:
Fixed in cl/738167833 and https://github.com/googleapis/python-storage/pull/1445/commits/77a3dc88e8ab2ce58a342321b1b4adc979afb0f7